### ISSUE TITLE
Operator pending mode no longer works with highlight_on_key

### ISSUE THREAD

**Issue by jemag** [jinh0/eyeliner.nvim#29]
Using commit 6a9d0ad714f11cab1691e54e936eaa9881cad277, operating pending like `dt{letter}` or `df{letter` no longer works with `  highlight_on_key = true`

Works fine if I instead pin eyeliner to commit e1df20beed2560914492db95ee7675d44c4ce96f

Config: 
```lua
require("eyeliner").setup({
  highlight_on_key = true,
  debug = false,
})
vim.api.nvim_create_autocmd({ "ColorScheme" }, {
  pattern = "*",
  callback = function()
    vim.api.nvim_set_hl(0, "EyelinerPrimary", { fg = "#ff00ff", bold = true, underline = false })
    vim.api.nvim_set_hl(0, "EyelinerSecondary", { fg = "#00ffff", underline = false })
  end,
})

```

**Comment by jinh0** [comment]
Hey @jemag , sorry about that. I just fixed it in the latest commit 3658815ed82b9d3c2e436d61c2e7f7d4aa72d307 . 

Seems that `nvim_feedkeys` doesn't like operator mode when given the option `ni` (which allows use of eyeliner in macros). Macros should work fine on the latest commit regardless of operator mode as well.

Thanks for submitting the issue!

**Comment by jemag** [comment]
Works great, thanks
